addons/crm/wizard/crm_lead_pls_description.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+from odoo.exceptions import UserError
+from odoo.tools.translate import _
+from odoo.tools.misc import _humanize_number
+
+
+class LeadPlsDescriptionField(models.TransientModel):
+    _name = 'crm.lead.pls.description.field'
+    _description = 'Information of the field used in the PLS calculation'
+
+    lead_pls_description_id = fields.Many2one('crm.lead.pls.description', readonly=True)
+
+    name = fields.Char('Name of the field', readonly=True)
+    value = fields.Char('String value of the field', readonly=True)
+
+    number_of_won = fields.Char('#Won', help='Number of won leads which have this value', readonly=True)
+    number_of_lost = fields.Char('#Lost', help='Number of lost leads which have this value', readonly=True)
+    total_won = fields.Char('Total Won', help='Total number of won leads', readonly=True)
+    total_lost = fields.Char('Total Lost', help='Total number of lost leads', readonly=True)
+
+    percent_won = fields.Integer('%Won', help='Percentage of lead that has this value and are won on all won leads', readonly=True)
+    percent_lost = fields.Integer('%Lost', help='Percentage of lead that has this value and are lost on all lost leads', readonly=True)
```

Comment:
@dbeguin ``won_total`` and ``lost_total`` should not be in ``crm.lead.pls.description`` because it depends on the field.
This attribute is the total number of won lead with the field value.
Maybe the help string should be more explicit